recipes/configure.rb

```diff
@@ -12,3 +12,31 @@
   group 'root'
   mode '0755'
 end
+
+hazelcast_xml = node['nexus_repository_manager']['nexus_home']['path'] + '/etc/fabric/hazelcast.xml'
+
+template hazelcast_xml do
+  source 'hazelcast.xml.erb'
+  owner 'root'
+  group 'root'
+  mode '0755'
+end
+
+nexus_properties = node['nexus_repository_manager']['nexus_home']['path'] + '/etc/nexus-default.properties'
+
+template nexus_properties do
+  source 'nexus.properties.erb'
+  owner 'nexus'
+  group 'nexus'
+  mode '0755'
+end
+
+ruby_block "remove java memory settings from nexus.vmoptions" do
```

Comment:
IIUC we are now removing this for every VM configured via chef-solo? What are the implications for AWS? Or any other VM? If it should be removed why is it there to begin with?

Reply:
Good catch! Moved this into the docker.rb recipe (123e54e).

Reply:
With regard to why it is there in the first place, it does not appear to be there anymore. I believe it is there in case someone added those settings so they do not conflict/override the INSTALL4J environment variable.

Reply:
I was wrong, the memory settings are in nexus.vmoptions.  I was looking at the after version of it. They are being removed so the INSTALL4J parameter settings will be used.